client/flutter/assets/content_bundles/symptom_checker.en.yaml

```
@@ -142,3 +175,280 @@ contents:
           icon_name: snowflake
           body_html: |
             <p>No</p>
+  cards:
```

Do we want to name this to be more clear what the cards are used for? E.g.:

```suggestion
  results_cards:
```